### PR TITLE
Cleanup TDx v1.0 (content from WZDx v4.0 Road Restriction Feed)

### PR DESCRIPTION
**Pull request by mark-mockett**
This PR cleans up the supporting documentation of the TDx repository to reflect a "v1.0" based on the WZDx v4.0 Road Restriction Feed. Recently approved modifications to the Road Restriction Feed and addition of a Road Incident Feed will comprise TDx v1.1.

### CODE REVIEW

File: Creating_a_TDx_Feed.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+2. If the `lanes` property on the [RestrictionRoadEvent](/spec-content/objects/RestrictionRoadEvent.md) is provided, it must include one entry for every lane in the road event. Providing lane information for only some of the lanes in a road event is not allowed.
+3. A [Lane](/spec-content/objects/Lane.md) `order` value of 1 must represent the left-most lane and an increase in 1 must represent moving a single lane to the right.
+4. The `data_source_id` value must match to the `data_source_id` property of a [FeedDataSource](/spec-content/objects/FeedDataSource.md) included within the same TDx GeoJSON document on the [RestrictionRoadEvent](/spec-content/objects/RestrictionRoadEvent.md)s.
+5. All dates and times must be expressed in UTC.
+
+## Implementation Guidance
+
+- Feed producers should include a completed road event with a verified start and end date in a public feed for a at least an hour or one feed refresh cycle (whichever is longer) to help inform consumers that the work zone has officially ended.
```
Comment (mollybehan) [inline Creating_a_TDx_Feed.md:36]: Remove extra "a". ...in a public feed for ### **a** at least an hour...

File: README.md [hunk 2]
```diff
@@ -1,128 +1,106 @@
...
+## Repository Organization
+The TDx Specification repository contains several files and subdirectories.
 
 ### Directories
-1. [documents](/documents): supplementary PDF and Word documents such as the WZDx Early Adopter's Guide and WZDx Data Feed Self Validation Checklist.
-2. [examples](/examples): example GeoJSON documents from WZDx data feeds. [examples/README.md](/examples/README.md) describes the content of this directory in detail.
-3. [images](/images): the images that are referenced by other Markdown files in the repository.
-4. [schemas](/schemas): contains [JSON Schema](https://json-schema.org/)s for each of the feeds defined by WZDx for feed validation.
-5. [spec-content](/spec-content): details the data content of the WZDx specification, including objects, property names and types, and enumerated types. [spec-content/README.md](/spec-content/README.md) describes the content of this directory in detail.
+1. [examples](/examples): example GeoJSON documents from TDx data feeds. [examples/README.md](/examples/README.md) describes the content of this directory in detail.
```
Comment (mollybehan) [inline README.md:30]: Should these be numbered 1. - n. instead of all "1." ?

Comment (mark-mockett) [reply README.md:30]: All 1s is okay - Markdown automatically renders it as a sequential numbered list. Using all 1s makes it easier to add something to the list without manually renumbering everything.

File: Creating_a_TDx_Feed.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Creating a TDx Feed
+This documents contains information to assist in creating a TDx data feed, such as the feed format, business rules, and validation tools.
+
+## Feed Format and File Type
+TDx feeds are formatted according to the [GeoJSON](https://geojson.org/) specification. The output of a TDx feed is a GeoJSON document (a `.geojson` file) that contains a single [GeoJSON FeatureCollection](https://datatracker.ietf.org/doc/html/rfc7946#section-3.3) which includes information about the feed (see the [FeedInfo Object](/spec-content/objects/FeedInfo.md)) and a list of [GeoJSON Feature](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2)s describing entities specific to the type of feed, such as work zones.
```
Comment (mollybehan) [inline Creating_a_TDx_Feed.md:5]: Should this section reference some kind of restriction instead of work zones? (last sentence)

**Review by mollybehan — APPROVED** [review]

**Pull request merged**